package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redis-dist-job-queue",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Sorry about that: want me to remove it before merge, or do you want to do it yourself?